codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/endpoints/EndpointParameterOperationBindingsGenerator.java

```diff
@@ -103,33 +100,14 @@ private GoWriter.Writable generateOperationContextParamBindings() {
     }
 
     private GoWriter.Writable generateOpContextParamBinding(String paramName, OperationContextParamDefinition def) {
-        var param = rules.getParameters().get(Identifier.of(paramName)).get();
         var expr = JmespathExpression.parse(def.getPath());
 
         return writer -> {
             var generator = new GoJmespathExpressionGenerator(ctx, writer);
 
             writer.write("func() {"); // contain the scope for each binding
             var result = generator.generate(expr, new GoJmespathExpressionGenerator.Variable(input, "in"));
-
-            if (param.getType().equals(ParameterType.STRING_ARRAY)) {
-                // projections can result in either []string OR []*string -- if the latter, we have to unwrap
```

Comment:
For context: discussed offline and this is done because projections don't have nulls, so they don't have `*string` members